### PR TITLE
[123] Fixing broken import from uds module

### PR DESCRIPTION
**Pull request by n135c10r**
Its fixing [this](https://github.com/CaringCaribou/caringcaribou/issues/123) issue.

### CODE REVIEW

**Review by kasperkarlsson — CHANGES_REQUESTED** [review]
The suggested change passes the whole response to the `print_negative_response_code` rather than the (integer) NRC from the response, which will lead to a runtime error 😅

**Comment by n135c10r** [comment]
Ok, I think it should work now 🙈 
Maybe there should be an additional function for retrieving the NRC code from the response?

File: caringcaribou/modules/uds_fuzz.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ def seed_randomness_fuzzer(args):
                         time.sleep(inter)
 
                     else:
-                        print_negative_response(response)
+                        # Negative response handling
+                        nrc = response[2]
+                        print_negative_response_code(nrc)
```
Comment (kasperkarlsson) [inline caringcaribou/modules/uds_fuzz.py:88]: Yep - this looks great! 👍

**Review by kasperkarlsson — APPROVED** [review]
This fixes the issue! Just as you say, adding a function to the `uds` module which retrieves the NRC from a message would be a good idea.

**Pull request merged**